docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -626,38 +636,35 @@ spec:
   - openstack
 EOF
 ----
-
-Note: if you have other node sets to deploy (e.g. for networker nodes), you may
-add them above in `nodeSets` list, or create separate
-`OpenStackDataPlaneDeployment` CRs later.
-
-Note that once a `OpenStackDataPlaneDeployment` CR is deployed, it's impossible
-to add new node sets to it later.
++
+[NOTE]
+If you have other node sets to deploy, for example, for networker nodes, you can
+add them in the `nodeSets` list in this step, or create separate `OpenStackDataPlaneDeployment` CRs later. Note that once a `OpenStackDataPlaneDeployment` CR is deployed, you cannot add new node sets to it later.
```

Comment:
"If you have other node sets to deploy, such as networker nodes, you can add them in the `nodeSets` list in this step, or create separate `OpenStackDataPlaneDeployment` CRs later. Note that after a `OpenStackDataPlaneDeployment` CR is deployed, you cannot add new node sets to it later."

change "once" to "if", "when", or "after"